benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json

```diff
@@ -2,7 +2,11 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "TabID": "summary_sscs", "TabLabel": "Summary", "TabDisplayOrder": 1, "CaseFieldID": "summaryRole", "FieldShowCondition": "appeal.appellant.role.name!=\"\"", "TabFieldDisplayOrder": 14},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "TabID": "summary_sscs", "TabLabel": "Summary", "TabDisplayOrder": 1, "CaseFieldID": "summaryRoleDescription", "FieldShowCondition": "appeal.appellant.role.description!=\"\"", "TabFieldDisplayOrder": 15},
   {"LiveFrom": "16/09/2019", "CaseTypeID": "Benefit", "TabID": "summary_sscs", "TabLabel": "Summary", "TabDisplayOrder": 1, "CaseFieldID": "pcqId", "TabFieldDisplayOrder": 31},
-  {"LiveFrom": "01/10/2021", "CaseTypeID": "Benefit", "TabID": "summary_sscs", "TabLabel": "Summary", "TabDisplayOrder": 1, "CaseFieldID": "summaryConfidentialityRequired", "FieldShowCondition": "appeal.appellant.confidentialityRequired!=\"\"", "TabFieldDisplayOrder": 32}
+  {"LiveFrom": "01/10/2021", "CaseTypeID": "Benefit", "TabID": "summary_sscs", "TabLabel": "Summary", "TabDisplayOrder": 1, "CaseFieldID": "summaryConfidentialityRequired", "FieldShowCondition": "appeal.appellant.confidentialityRequired!=\"\"", "TabFieldDisplayOrder": 32},
+  {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "TabID": "caseInfo_sscs", "TabLabel": "Appeal Details", "TabDisplayOrder": 4, "CaseFieldID": "isFqpmRequired", "TabFieldDisplayOrder": 38, "FieldShowCondition": "isFqpmRequired!=\"\""},
```

Comment:
This line doesn't belong here please move it down one line to make the job the person releasing it a bit easier.

Reply:
all lines at the bottom are grouped per feature, can't see why you think it doesn't belong there.

Reply:
It was not grouped per tab id. It's not very interesting to explain. Before this project we had to update a spreadsheet that regularly got into a mess.